### PR TITLE
Feature/jellyfish availability alert

### PR DESCRIPTION
**Pull request by julianzimmermann**
added jellyfish support

### CODE REVIEW

File: bundles/jellyfish-availability-alert/src/FondOfOryx/Zed/JellyfishAvailabilityAlert/Business/Model/Dispatcher/AvailabilityAlertSubscriberDispatcher.php [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+            $this->getLogger()->error(
+                sprintf(
+                    'AvailabilityAlertSubscriber with id "%s" could not be dispatched to JellyFish! Message: %s',
+                    $availabilityAlertSubscriberTransfer->getIdAvailabilityAlertSubscriber(),
+                    $exception->getMessage()
+                ),
+                $exception->getTrace()
+            );
+
+            throw $exception;
```
Comment (sourcecube) [inline bundles/jellyfish-availability-alert/src/FondOfOryx/Zed/JellyfishAvailabilityAlert/Business/Model/Dispatcher/AvailabilityAlertSubscriberDispatcher.php:49]: I think catch and instant throw is unnecessary

**Review by sourcecube — COMMENTED** [review]
Some files need general formatting

File: bundles/jellyfish-availability-alert/src/FondOfOryx/Zed/JellyfishAvailabilityAlert/Business/Model/Dispatcher/AvailabilityAlertSubscriberDispatcher.php (hunk 1 above)
Comment (julianzimmermann) [reply bundles/jellyfish-availability-alert/src/FondOfOryx/Zed/JellyfishAvailabilityAlert/Business/Model/Dispatcher/AvailabilityAlertSubscriberDispatcher.php:49]: needed for the logging

**Review by daniel-rose — COMMENTED** [review]
Checks are failed, because _bootstrap.php is missing in jellyfish-availability-alert.

File: bundles/jellyfish-availability-alert/composer.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
+{
+  "name": "fond-of-oryx/jellyfish-availability-alert-",
```
[the remark this answers is not in the document]
Comment (daniel-rose) [reply bundles/jellyfish-availability-alert/composer.json]: Typo?

File: bundles/jellyfish-availability-alert/codeception.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
+namespace: fond-of-oryx/jellyfish-availability-alert-
```
[the remark this answers is not in the document]
Comment (daniel-rose) [reply bundles/jellyfish-availability-alert/codeception.yml]: Typo?

File: bundles/jellyfish-availability-alert/README.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+#  Module
+[![PHP from Travis config](https://img.shields.io/travis/php-v/symfony/symfony.svg)](https://php.net/)
+[![license](https://img.shields.io/github/license/mashape/apistatus.svg)](https://packagist.org/packages/fond-of-oryx/jellyfish-availability-alert-)
+
+## Installation
+
+```
+composer require fond-of-oryx/jellyfish-availability-alert-
```
[the remark this answers is not in the document]
Comment (daniel-rose) [reply bundles/jellyfish-availability-alert/README.md]: Typo?

File: bundles/jellyfish-availability-alert/.gitignore [hunk 5]
```diff
@@ -0,0 +1,6 @@
+.idea
+
+.DS_Store
+
+vendor
+composer.lock
```
Comment (daniel-rose) [inline bundles/jellyfish-availability-alert/.gitignore:6]: Generated folder is missing...

File: bundles/jellyfish-availability-alert/README.md [hunk 6]
```diff
@@ -0,0 +1,9 @@
+#  Module
+[![PHP from Travis config](https://img.shields.io/travis/php-v/symfony/symfony.svg)](https://php.net/)
```
Comment (daniel-rose) [inline bundles/jellyfish-availability-alert/README.md:2]: Remove travis stuff...

File: bundles/jellyfish-availability-alert/README.md [hunk 7]
```diff
@@ -0,0 +1,9 @@
...
+[![license](https://img.shields.io/github/license/mashape/apistatus.svg)](https://packagist.org/packages/fond-of-oryx/jellyfish-availability-alert)
```
Comment (daniel-rose) [inline bundles/jellyfish-availability-alert/README.md:3]: https://img.shields.io/github/license/mashape/apistatus.svg point to wrong package "mashape/apistatus"

File: bundles/jellyfish-availability-alert/composer.json [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+  "description": "n/a",
+  "license": "MIT",
+  "authors": [
+    {
+      "name": "Julian Zimmermann",
+      "email": "info@julianzimmermann.dev"
+    }
+  ],
+  "require": {
+    "php": ">=7.2",
```
Comment (daniel-rose) [inline bundles/jellyfish-availability-alert/composer.json:12]: 7.2 is end of life. Use always >=7.3

File: bundles/jellyfish-availability-alert/composer.json [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+    "fond-of-spryker/jellyfish": "dev-feature/refractoring-remove-order-export"
+  },
+  "require-dev": {
+    "spryker/code-sniffer": "^0.14.7",
```
Comment (daniel-rose) [inline bundles/jellyfish-availability-alert/composer.json:16]: use "spryker/code-sniffer": "^0.14"

File: bundles/jellyfish-availability-alert/composer.json [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+    "codeception/codeception": "^2.5",
+    "mikey179/vfsstream": "^1.6",
+    "phpro/grumphp": "^0.14",
+    "sebastian/phpcpd": "^4.1",
+    "fond-of-codeception/spryker": "dev-master",
```
Comment (daniel-rose) [inline bundles/jellyfish-availability-alert/composer.json:21]: Use fix version...

File: bundles/jellyfish-availability-alert/composer.json [hunk 11]
```diff
@@ -0,0 +1,37 @@
...
+  "prefer-stable": true,
+  "autoload": {
+    "psr-4": {
+      "FondOfOryx\\": "src/FondOfOryx"
+    }
+  },
+  "autoload-dev": {
+    "psr-4": {
+      "FondOfOryx\\": "tests/FondOfOryx",
+      "Generated\\": "tests/Generated"
```
Comment (daniel-rose) [inline bundles/jellyfish-availability-alert/composer.json:34]: The generated stuff can you find under src/Generated...

**Review by daniel-rose — CHANGES_REQUESTED** [review]

File: bundles/jellyfish-availability-alert/codeception.yml (hunk 3 above)
[the remark this answers is not in the document]
Comment (julianzimmermann) [reply bundles/jellyfish-availability-alert/codeception.yml]: nah search and replace I have renamed the module :)

File: bundles/jellyfish-availability-alert/.gitignore (hunk 5 above)
Comment (julianzimmermann) [reply bundles/jellyfish-availability-alert/.gitignore:6]: Not even a single module in foo namespace has Generated folder in .gitignore

**Review by sourcecube — APPROVED** [review]

**Pull request merged**